manual/manual/cmds/intf-c.etex

```diff
@@ -2702,3 +2698,67 @@ support libraries ("-lz") and the corresponding options
 ("-L/usr/local/zlib") must be given on all three invocations of "ocamlmklib",
 because they are needed at different times depending on whether shared
 libraries are supported.
+
+
+\section{s:c-internal-guidelines}{Disavowed topic: the internal runtime API}
+
+Let's start with a warning, there is \emph{no} stability guarantee if you are
+using internals of the OCaml runtime. If you really need access
+to the runtime internal API, this section provides some guidelines
+that may help you to write code that might not break on every new version
+of OCaml.
+
+\subsection{ss:c-internals}{CAML_INTERNALS}
+First, by default header files in the "caml/" subdirectory give only access
+to the public interface of the OCaml runtime.
+It is necessary to define the macro "CAML_INTERNALS" to get access to
+the lower-level interface. Once again, this lower-level interface is likely to
+change and break programs that use it.
```

Comment:
"s/likely to change/may change with every minor release" to make it explicit that there is really no guarantee.

Reply:
Good idea. Done.